### DISCUSSION TITLE
Allow roll result to be changed from chat messages (reroll, Success to Failure, etc)

### DISCUSSION THREAD

**Discussion by Savantford** [Savantford/foundry-weirdwizard#275] [New Feature Requests]
"Many abilities/spells have the ability to turn a failure into a success (a few examples I came across are Kill Everyone, Inspiring Command, and All-Consuming Fires of Hell). Since there are different things shown in chat based on the success of the roll, can you add an option to change the success state of something in chat?

ie, turns misses into hits so you can roll damage"

"When the fighter gets a critical he gets to roll damage twice and keep the highest.
I think this will also be added to the reroll request
reroll/turn success into failure/etc
basically button functionality from outputted chat cards"
